configuration/index.md

```diff
@@ -99,43 +102,65 @@ Every interface has it's own strong points.
     </tr>
     <tr>
       <td>Define Rules</td>
-      <td>✔️</td>
-      <td>✔️<br />(Experimental)</td>
-      <td>✔️<br />(Graphical)</td>
+      <td>✔️<br/>(recommended)</td>
+      <td>(✔️)<br/>(experimental)</td>
+      <td>(❌)<br/>(not functional)</td>
       <td>❌</td>
     </tr>
     <tr>
-      <td>Manage Z-Wave Devices</td>
-      <td>✔️<br />(Not&nbsp;Recommended)</td>
-      <td>✔️<br />(Limited)</td>
-      <td>✔️</td>
+      <td>Manage Z-Wave Devices<br/>(see&nbsp;recommendation)</td>
+      <td>❌</td>
+      <td>✔️<br />(limited)</td>
+      <td>✔️<br />(recommended)</td>
       <td>❌</td>
     </tr>
     <tr>
       <td>Modify openHAB Settings</td>
       <td>✔️</td>
+      <td>✔️<br/>(recommended)</td>
       <td>✔️</td>
-      <td>✔️</td>
-      <td>✔️</td>
+      <td>(✔️)</td>
     </tr>
     <tr>
       <td>Install Add-ons</td>
       <td>✔️</td>
+      <td>✔️<br/>(recommended)</td>
       <td>✔️</td>
-      <td>❌</td>
       <td>✔️</td>
     </tr>
   </tbody>
 </table>
 
-{: #text-vs-ui}
 ### Textual vs. Graphical Configuration
 
-With openHAB 1.x the smart home configuration was done via configuration files only.
-One of the most visible additions to openHAB 2 is the administrative web interface Paper UI.
+In openHAB 1.x, smart home configuration was done via configuration files only.
+openHAB 2 added the administrative web interface "Paper UI".
+It allows for configuration of the base system and setup of openHAB 2 add-ons.
+Please be aware that not every addon/binding of openHAB version 1.x has been updated to version 2 and thus to work with Paper UI.
+You can still run these but you need to use file configuration for them.
 
 Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database]({{base}}/administration/jsondb.html).
-
 Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
-
-Configuration done in files will be synced to the database but configuration done in the database will not be written to files.
+Configuration done in files will be synced to the database (so they are accessible in Paper UI) but configuration done in the database will not be written back to files, so if you restart openHAB, changes made in Paper UI won't survive and the files' configuration data will be used again.
+
+## Recommendation for New Users
```

Comment:
All of the bullets are sentences, but some are missing punctuation and/or capitalization.

Reply:
huh? what bullets, what missing punctuation?

Reply:
parentheses were to indicate to not use it cause it's cumbersome and somewhat limited, but as that's not that easy to understand, I chose to remove them.
Changed the rest according to your comments.